src/models/cases.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+module.exports = CasesFactory
+
+const co = require('co')
+const CaseFactory = require('./db/case')
+const NotaryFactory = require('./db/notary')
+const Log = require('../lib/log')
+const DB = require('../lib/db')
+const Config = require('../lib/config')
+const NotificationWorker = require('../lib/notificationWorker')
+const CaseExpiryMonitor = require('../lib/caseExpiryMonitor')
+const Condition = require('five-bells-condition').Condition
+const UnprocessableEntityError = require('five-bells-shared').UnprocessableEntityError
+const UnmetConditionError = require('five-bells-shared').UnmetConditionError
+const NotFoundError = require('five-bells-shared').NotFoundError
+
+CasesFactory.constitute = [CaseFactory, NotaryFactory, Log, DB, Config, NotificationWorker, CaseExpiryMonitor]
+function CasesFactory (Case, Notary, log, db, config, notificationWorker, caseExpiryMonitor) {
+  log = log('cases')
```

Comment:
Should avoid reassigning parameters. Could just rename the parameter to "logger".